inspire_classifier/cli.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2019 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+import click
+import click_spinner
+from flask import current_app
+from flask.cli import FlaskGroup, with_appcontext
+from inspire_classifier.api import (
+    train,
+    predict_coreness
+)
+from inspire_classifier.app import create_app
+
+
+@click.group(cls=FlaskGroup, create_app=create_app)
+def inspire_classifier():
+    "INSPIRE Classifier commands"
+
+
+@inspire_classifier.command('predict-coreness')
+@with_appcontext
+@click.argument('title', type=str, required=True, nargs=1)
+@click.argument('abstract', type=str, required=True, nargs=1)
+@click.option('-b', '--base_path', type=click.Path(exists=True), required=False, nargs=1)
```

Comment:
usually `-` are used instead of `_` in option names. I think click takes care of the conversion automatically though.